static/app/components/globalDrawer/types.tsx

```diff
@@ -0,0 +1,46 @@
+import type {DrawerBody} from 'sentry/components/globalDrawer/components';
+
+export interface DrawerOptions {
+  /**
+   * If true (default), closes the drawer when an escape key is pressed
+   */
+  closeOnEscapeKeypress?: boolean;
+  /**
+   * If true (default), closes the drawer when anywhere else is clicked
+   */
+  closeOnOutsideClick?: boolean;
+  /**
+   * Callback for when the drawer closes
+   */
+  onClose?: () => void;
+  /**
+   * Callback for when the drawer opens
+   */
+  onOpen?: () => void;
```

Comment:
Do we even need this? Don't they have to call `openDrawer` anyway, can't they just call whatever they would pass to that right after they call `openDrawer`?